migrations/20210422071821_add_id_primary_key.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema
+        .alterTable('newsletters', (table) => {
+          table.string('year_week').nullable().alter();
+          table.increments('id').primary().unsigned();
```

Comment:
On utilise des UUIDs par défaut, c'est ce que l'on recommande pour toutes les apps sauf s'il y a une justification contraire.
Dans ce cas là, ça passe, il n'y a pas beaucoup d'enjeux mais je me dis autant pratiquer les UUIDs pour prendre des bonnes habitudes

Reply:
UUID it is